renderer/components/Settings/SettingsPage.js

```diff
@@ -105,9 +105,14 @@ const SettingsPage = ({ currentConfig, ...rest }) => {
           <FormattedMessage {...messages.settings_title} />
         </Heading.h1>
 
-        <SettingsForm>
+        <SettingsForm onSubmit={fetchTickers}>
```

Comment:
I'd probably move this to `SettingsForm::handleSubmit` which is where we handle all the other special submit behaviour for the form.